appservice/src/createAppService/SiteCreateStep.ts

```diff
@@ -76,9 +76,15 @@ export class SiteCreateStep extends AzureWizardExecuteStep<IAppServiceWizardCont
                 fileShareName = `${fileShareName.slice(0, maxFileShareNameLength - randomLetters - 1)}-${randomUtils.getRandomHexString(randomLetters)}`;
             }
 
+            let endpointSuffix: string = wizardContext.environment.storageEndpointSuffix;
+            // https://github.com/Azure/azure-sdk-for-node/issues/4706
```

Comment:
nit: you could do something like:
endpointSuffix = endpointSuffix.replace(/^\./, '');

Reply:
Good thing I tested this because markdown apparently removed a slash 😬😬😬
Preview tab:
![screen shot 2019-02-22 at 11 55 10 am](https://user-images.githubusercontent.com/11282622/53267659-c0dabb80-3698-11e9-8d22-9c5e064e006e.png)

Write tab:
![screen shot 2019-02-22 at 11 55 15 am](https://user-images.githubusercontent.com/11282622/53267661-c0dabb80-3698-11e9-8372-6ac7edd05759.png)